### DISCUSSION TITLE
流量使用没有消耗

### DISCUSSION THREAD

**Discussion by hkh0324** [natfrp/wiki#552] [使用问题] [answered]
我用sakurafrp对饥荒联机版内网穿透，大概在半个月前我注意到只要打开软件隧道启动就会有流量消耗，但是今天再次打开启动隧道流量不管怎么样都纹丝未动，这是为什么，以及有什么影响
![0ed90cea81e1d0e7f29bdd448b0cb73f](https://github.com/user-attachments/assets/705a3670-a866-4f46-9e3e-a0140babd17e)

[a comment by an automated account is not shown]

**Reply by hkh0324** [reply]
感谢解答，我使用代码连接进游戏后流量开始正常使用了。不过我还有点疑惑，此前我和我的朋友都没有使用代码加入游戏而是直接在大厅加入游戏，但是这种情况也会有少量流量的使用，像图上一样仅仅是打开内网穿透就会有流量消耗，并且流量使用的很少，今天我刚加入就使用了大约是一个月前玩一整天消耗的流量的三倍，这是为什么

**Reply by hkh0324** [reply]
十分感谢您的回答

**Reply by pemo-x** [reply]
你好，想问一下你的饥荒开了几个隧道呀，我们开了两个，udp的10999主世界，dup的10998洞穴，但是下不去洞穴耶
然后尝试按照帮助文档里写的要把存档的server.ini修改成隧道的端口，但启动游戏后会自动修改回10999和10998，就很迷茫...

**Reply by hkh0324** [reply]
> 你好，想问一下你的饥荒开了几个隧道呀，我们开了两个，udp的10999主世界，dup的10998洞穴，但是下不去洞穴耶 然后尝试按照帮助文档里写的要把存档的server.ini修改成隧道的端口，但启动游戏后会自动修改回10999和10998，就很迷茫...

我不难能十分确定你的问题到底出在哪里。我开了四个隧道，因为我开了海难mod，四个世界对应四个隧道，我猜想如果你只是按照帮助文档简单得把存档里的server.ini的端口改成你的隧道端口实际上是不可行的，因为从客户端直接进入的存档世界他的端口固定是10999和10998，所有你尽管改了也不会有用。你需要创建一个专用服务器，专用服务器里对应的每个世界server.ini里的端口是可以更改的，如果你实在疑惑不解，https://www.bilibili.com/video/BV1wT4y1h7Za/?vd_source=e5c52fef1e1dc7489f73cec9d992630a#reply923949644这个网址的视频是我本人制作的视频可能能给你一些帮助，尽管有些过时了，如果你想知道最新最简单的办法可以直接在这个平台联系我，我可以教你。

[a comment by an automated account is not shown]
